service/src/test/java/uk/gov/hmcts/reform/fpl/service/validators/AllocationProposalCheckerIsStartedTest.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.fpl.service.validators;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.NullSource;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.fpl.model.Allocation;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(MockitoExtension.class)
+class AllocationProposalCheckerIsStartedTest {
+
+    @InjectMocks
+    private AllocationProposalChecker allocationProposalChecker;
+
+    @ParameterizedTest
+    @NullSource
+    @MethodSource("emptyAllocationProposal")
```

Comment:
Test passes even without this methodSource, i guess because of NullSource. So, do we need method source?

Reply:
same in couple of other tests..

Reply:
yes, we need. If MerhodSource is remove only null values will be checked, if MethodSource is there then nullsources and all combinations from emptyAllocationProposal will be tested

Reply:
aha ok. Good to know. Thanks for explaining 👍